webapp/src/pages/ResponseList/components/ResponseListForm.js

```diff
@@ -0,0 +1,251 @@
+import React, { Component } from 'react';
+import '../ResponseList.css';
+import { withTranslation } from 'react-i18next';
+import { fetchResponse, fetchQuestions } from '../../../services/ResponseList/ResponseList'
+import ReactTable from 'react-table';
+import "react-table/react-table.css";
+import ReactTooltip from 'react-tooltip';
+
+
+class ResponseListForm extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            questions: [],
+            selected: [],
+            toggleList: false,
+            responseTable: null,
+            addBtn: {
+                background: "hsl(211deg 100% 50%)",
+            }
+        }
+    }
+
+
+    componentWillMount() {
+        this.fetchData()
+    }
+
+
+    fetchData() {
+        fetchQuestions().then(response => {
+            this.setState({
+                questions: response
+            })
+        })
+    }
+
+
+    handleSelect(question) {
+        const selected = this.state.selected;
+        let duplicate = selected.indexOf(question)
+
+        if (duplicate == -1) {
+            selected.push(question)
+        }
+        else {
+            selected.splice(duplicate, 1)
+        }
+
+        this.setState({
+            selected: selected,
+            addBtn: { background: "hsl(134deg 61% 41%)" }
+        }, () => this.fetchData())
+    }
+
+
+    toggleList(list) {
+        this.setState({
+            toggleList: !list ? true : false
+        })
+    }
+
+
+    handleData() {
+        // disable question list
+        this.toggleList(true)
+
+        const { selected } = this.state;
+
+        // add Answers as new column 
+        function addAnswersCol(row, newAnswers) {
+            return row.answers = newAnswers
+        }
+        // add Reviews as new column 
+        function addReviewCol(row, newAnswers) {
+            return row.answers = newAnswers
+        }
+
+
+        fetchResponse().then(response => {
+            let handleAnwsers = [];
+            let handleReviews = [];
+            // Handle Anwsers and Reviews
+            response.map(val => {
```

Comment:
Since your lambda function has side-effects and doesn't return anything, you should use `forEach` instead of `map`